src/tox_ansible/ansible/scenario.py

```diff
@@ -32,10 +33,34 @@ def name(self):
     @property
     def driver(self):
         """Reads the driver for this scenario, if one is defined.
+        If there is a driver found in the scenario configuration and if there
+        is a global configuration, the driver coming from the scenario
+        will be returned. Otherwise, the global driver.
 
-        :return: Driver name defined in molecule.yml or None"""
+        :return: Driver name defined in molecule.yml or None
+        :raise: A RuntimeError if the driver name is present in multiple
+                molecule base configuration files given as options in the
+                tox.ini file. Or if no driver configuration has been found.
+        """
         if self.config and "driver" in self.config and "name" in self.config["driver"]:
             return self.config["driver"]["name"]
+
+        if self.global_config:
+            drivers_found_number = len(
+                [i for i, d in enumerate(self.global_config) if "driver" in d]
+            )
+            if 1 <= drivers_found_number < 2:
```

Comment:
Isn't this conditional just testing `drivers_found_number == 1`?

Reply:
Yes it is clearly that. I'll simplify it too.